src/index.ts

```diff
@@ -202,6 +202,11 @@ process.on("SIGUSR2", async () => await cleanupJobResources(jobs));
             description: "Sets docker user to 0:0",
             requiresArg: false,
         })
+        .option("userns", {
+            type: "string",
+            description: "Passes the argument to podmans --userns option",
```

Comment:
```suggestion
            description: "Set docker executor userns option",
```

Reply:
Would be OK for me, but AFAIK docker does not have the `userns` option, only podman does. I thought about adding a check for this option so it can only be combined with podman es executor, but did not see any other parameter being handled that way.

Reply:
```
mjn@mjn-lapdows:~$ docker run --help | grep userns
      --userns string                    User namespace to use
```

Reply:
Added your suggestion and addressed the [eslint finding](https://github.com/firecow/gitlab-ci-local/actions/runs/15021932826/job/42263206098).